bin/worker

```diff
@@ -14,7 +16,7 @@ job = ToolingInvoker::TestRunnerJob.new(
   "ruby",
   "bob",
   "s3://exercism-iterations/production/iterations/1182520",
-  nil, 
+  nil,
   nil
 )
 puts ToolingInvoker::InvokeLocally.(job)
```

Comment:
You can delete this, @joshgoebel.

Reply:
I could. Must I?  Until we have proper .editorconfigs in place if these are minimal can we just leave them?

Reply:
I mean the whole block (GH has failed to cope me clicking that). `exit` and everything down in this file.

Reply:
> $LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)

Do you have any idea what this is supposed to be doing? (line 4)

Reply:
Yeah - it loads the `lib` directory into the load path so that you can then do the require of `tooling_invoker` and the other files without having to use `require_relative`

Reply:
No - it's going TWO directories higher, that's outside the checkout completely... is it not?

Reply:
I don't think so. The first `..` is the file itself when in the context of `__FILE__`

Reply:
Ha, I think I was thinking of the more recent:

```
$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
```

You're right.